Examples/rc-maestro/maestro/e2e_tests/subscribe_from_v2_paywall.yaml

```diff
@@ -0,0 +1,21 @@
+appId: com.revenuecat.maestro.ios
+name: Subscribe to product through V2 Paywall for default offering
+
+---
+- launchApp:
+    arguments: 
+        e2e_test_flow: subscribe_from_v2_paywall
+- tapOn:
+    text: "Present Paywall"
+- assertVisible: "Paywall V2"
+- takeScreenshot: "subscribe_from_v2_paywall - Paywall"
+- tapOn:
+    text: "Yearly"
+- tapOn:
+    text: "Continue"
+- takeScreenshot: "subscribe_from_v2_paywall - IAP Sheet"
+- tapOn:
+    text: "Subscribe"
+- takeScreenshot: "subscribe_from_v2_paywall - IAP Confirmation"
+- tapOn:
+    text: "OK"
```

Comment:
Might be great to have a way to visualize "active_entitlements" in the UI, so we can verify the entitlements/purchase went through after purchasing. But this can be added later as well :)

Reply:
Yep, fully agree! Will be working on that right after